modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -55,8 +55,62 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Implement AsyncAPI Specifications
+
+Studio supports the implementation of AsyncAPI 2.6 specs. When creating an implementation project, Studio imports and scaffolds an AsyncAPI spec hosted on Exchange into an API interface that you can implement.
+
+For supported protocols, see xref:apikit::apikit-asyncapi-module-reference.adoc#supported-message-brokers[Supported Message Brokers] in APIkit for AsyncAPI Module Reference.
+
+=== Access AsyncAPI Implementation Features
+
+. Open Anypoint Code Builder landing page on your Anypoint Platform cloud host (https://anypoint.mulesoft.com/codebuilder/[US] or https://eu1.anypoint.mulesoft.com/codebuilder/[EU]). 
++
+Sign-in is required. The *Terms & Conditions* button is available to the organization administrator only.
+. If the message *AsyncAPI Beta is now available in Anypoint Code Builder and Anypoint Studio* appears on the page, ask your organization administrator to join the Beta program by:
+.. Clicking *Join Beta* from the Anypoint Code Builder landing page.
++
+The *Join Beta* button is available only to your organization administrator.
+.. Reviewing the terms and conditions of the beta program
+.. Clicking *Get Access*
++
+When the AsyncAPI feature is available for your organization, the Anypoint Code Builder landing page provides the following notification:
++
+*AsyncAPI Implementation Beta for Anypoint Code Builder and Studio is enabled for your organization.*
+
+[NOTE]
+You must have access to a supported AsyncAPI specification hosted on Anypoint Exchange. If your AsyncAPI specification is not published to Exchange, see xref:design-center::design-publish.adoc[]. To create an AsyncAPI specification before publishing to Exchange, see xref:design-center::design-async-api.adoc[] in the Design Center documentation.
+
+=== Scaffolding Fundamentals for AsyncAPI Implementations
+
+When scaffolding an AsyncAPI specification into a Mule project (implementation or integration), Studio:
+
+* Introspects the AsyncAPI specification
+* Creates a new Mule project with a separate flow for each `publish` operation in the specification
+* Produces the configuration properties `file dev-properties.properties` file in the `src/main/resources` directory of your Mule project
+* Makes the APIkit for AsyncAPI module available for use in the project
```

Comment:
```suggestion
* Makes the APIkit for AsyncAPI module available for use in the project
```
do we capitalize "module" in this case; compare with below

Reply:
We should not capitalize “module” since it's not part of the name of the product. It’s only its category.

Reply:
@Crispy-Salesforce Do you see how all our connector docs do this? One example: https://docs.mulesoft.com/http-connector/latest/

Reply:
Normally, I wouldn't capitalize it either. I'm wondering if we should conform here.

Reply:
To be consistent with the APIkit docs, the "module" world should not be capitalized.
If the "module" word should be capitalized, I can add such a task to my backlog to discuss about it later.
Thanks!